saleor/graphql/schema.graphql

```diff
@@ -4669,6 +4678,24 @@ type ShippingPriceUpdate {
   shippingErrors: [ShippingError!]!
 }
 
+type ShippingZipCodeCreate {
+  errors: [Error!]! @deprecated(reason: "Use typed errors with error codes. This field will be removed after 2020-07-31.")
+  shippingMethodZipCode: ShippingMethodZipCode
```

Comment:
Rest of the mutations follow the rule that they should return actual object that was modified - in this case it would be either `ShippingMethod` or `ShippingZone`